src/pages/Home.js

```diff
@@ -75,7 +77,7 @@ export default function Home(props) {
             onChange={handleInputValue}
           />
         </label>
-        <input type="submit" value="Submit" />
+        <IconButton type="submit" text=">" />
```

Comment:
When a screen reader gets to `>`, would it read it as "greater than"?

Reply:
Eventually we will reference #34 and include the relevant [Heroicons](https://heroicons.com/) in place of the `X` and `>`. But I used those to visually represent what the Figma design currently has in its place. 

I'm trying NOT to combine too many issues into a chunk of work or PR.